File: src/main/java/org/jukeboxmc/potion/ResistanceEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ResistanceEffect extends Effect {

    @Override
    public int getId() {
        return 11;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.RESISTANCE;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 153, 69, 58 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/ResistanceEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class ResistanceEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 11
    }

    override fun getEffectType(): EffectType {
        return EffectType.RESISTANCE
    }

    override fun getEffectColor(): Color {
        return Color(153, 69, 58)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}